cosmos-auth/conf/cosmos-auth.json

```diff
@@ -1,6 +1,8 @@
 {
```

Comment:
I had it, but forgot to push it :dancer: 
Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/0cb37135d3acfa15c75582c4c0a86604495f2265